domain-sample-data/build.gradle

```diff
@@ -8,4 +8,8 @@ repositories {
 
 dependencies {
   compile project(':domain-model')
+  compile project(':ccd-adapter')
+  compile group: 'uk.gov.hmcts.reform', name: 'core-case-data-store-client', version: '4.6.2'
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.9.9'
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.9.9.3'
```

Comment:
we are using jackson version `2.10.0` in our main module. please use the same here as well.

Reply:
done